### ISSUE TITLE
NVDA cannot interact with the Viber Interface.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3961]
**Reported by mcigor17 on 2014-03-09 00:12**
When attempting to navigate Viber's interface, NVDA does not report any objects under the system focus or the review cursor. The only thing NVDA reports is "Viber" when using the title command.

**Comment by ehollig** [comment]
As @jcsteh said in https://github.com/nvaccess/nvda/issues/3961#issuecomment-155318134, please contact the developers of Viber to make this program more accessible for screen readers. Closing

**Comment by AgentGolder** [comment]
Hello all, hope you are fine.
Last week I contacted them, they asked me some elementary questions, I answered all of them and they just toled me that Viber is no longer supported on Windows 7. Anyone who runs Windows 10 (64 bit) or 11 can try to see if [nvda+r] works to make it accessible using OCR. What should I tell them to take care? Any of you who is also interested can contact them using the following form:
[https://help.viber.com/en/contact](url)
E-mail attached
[Viber accessibility.txt](https://github.com/nvaccess/nvda/files/11015008/Viber.accessibility.txt)
.
